requirements-dev.txt

```diff
@@ -76,8 +76,10 @@ iniconfig==2.0.0          # via -c constraints.txt, pytest
 ipykernel==6.29.4         # via -c constraints.txt, nbmake
 ipython==8.12.3           # via -c constraints.txt, ipykernel
 isort==5.13.2             # via -c constraints.txt, -r requirements-dev.in
-jax[cpu]==0.4.13          # via -c constraints.txt, gt4py (pyproject.toml)
-jaxlib==0.4.13            # via -c constraints.txt, jax
+jax==0.4.13 ; python_version < "3.9"  # via -c constraints.txt, gt4py (pyproject.toml)
+jax==0.4.18 ; python_version >= "3.9"  # via -c constraints.txt, gt4py (pyproject.toml)
+jaxlib==0.4.13 ; python_version < "3.9"  # via -c constraints.txt, jax
+jaxlib==0.4.18 ; python_version >= "3.9"  # via -c constraints.txt, jax
```

Comment:
Was this a manual change?  It will be overwritten in the next run of updating dependencies.
Probably need something like
```
jax-cpu = ['jax[cpu]>=0.4.13,<0.4.18;python_version<"3.9"', 'jax[cpu]>=0.4.18; python_version>="3.9"']
```
in pyproject.toml, but the update script will not propagate both entries properly.

Reply:
Actually we don't need jax for python < 3.10, so we should add 

```
jax-cpu = ['jax[cpu]>=0.4.13; python_version>="3.10"']
```
(or whatever other minimum version we need in jax)

Reply:
Done, thank you.